core/src/main/java/org/bitcoinj/script/ScriptChunk.java

```diff
@@ -70,6 +71,10 @@ public boolean isPushData() {
         return opcode <= OP_16;
     }
 
+    public int getDataLength() {
```

Comment:
This public method is missing a JavaDoc.

Is it a good idea to return 0 if the chunk is no data? I'd expect `null`.

Reply:
The idea was to avoid checking for null data when pattern matching. In order to absolutely avoid any future problems, I will remove this and just change the pattern matching code.